consensus/istanbul/backend/snapshot.go

```diff
@@ -243,6 +243,15 @@ func (s *Snapshot) validators() []common.Address {
 	return sortValidatorArray(validators)
 }
 
+// demotedValidators retrieves the list of authorized but demoted validators in ascending order.
```

Comment:
What about adding comma like `GetDemotedValidatorsAtHash` comment?
```suggestion
// demotedValidators retrieves the list of authorized, but demoted validators in ascending order.
```